src/Exception/TufException.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Tuf\Exception;
+
+/**
+ * Defines a base class for all exceptions thrown by php-tuf.
+ */
+abstract class TufException extends \Exception
```

Comment:
Let's see how this goes with it abstract. I might want to change it back but I'll cite a specific use case if that happens.